tests/foreman/api/test_hostgroup.py

```diff
@@ -70,21 +70,16 @@ def test_bz_1107708(self):
         # Get all environments for current organization.
         # We have two environments (one created after publishing and one more
         # was created after promotion), so we need to select promoted one
-        response = client.get(
-            entities.Environment().path(),
-            auth=get_server_credentials(),
-            data={u'organization_id': org.id},
-            verify=False,
+        environments = entities.Environment().search(
+            query={'organization_id': org.id}
```

Comment:
This is an odd case. An `Environment` can be associated with multiple organizations. As a result, this search is generated by default:

```python
>>> org = entities.Organization(id=1)
>>> env = entities.Environment(organization=[org])
>>> env.search_payload()
{'organization_ids': [1]}
```

But the server only accepts a single ID when a search is performed. So we have to manually put together a query.